trailer.js

```diff
@@ -272,6 +272,13 @@ async function ngrokForward(config) {
     }
     config = { addr: address };
   }
+  if (typeof config["port"] === "string" || config["port"] instanceof String) {
+    const num = parseInt(config["port"]);
```

Comment:
it might be worth throwing a radix `10` on this. That'll take things like `0X50` to `0` instead of `80`.